packages/fast-tooling-react/src/form/templates/template.control.utilities.tsx

```diff
@@ -32,7 +32,7 @@ abstract class ControlTemplateUtilities<P, S> extends React.Component<
         }
     }
 
-    public renderSoftRemove(className: string): React.ReactNode {
+    public renderSoftRemove(className: string): JSX.Element {
```

Comment:
can we change this file back? we may be exporting it at some point and we want to be as permissive as possible, it shouldn't error.

Reply:
Tried this out - reverting doesn't cause issues.